be/src/exec/vectorized/chunks_sorter_topn.h

```diff
@@ -11,18 +11,33 @@ namespace starrocks::vectorized {
 // Sort Chunks in memory with specified order by rules.
 class ChunksSorterTopn : public ChunksSorter {
 public:
+    static constexpr size_t kMaxBufferedChunks = 512;
+    static constexpr size_t kMinBufferedChunks = 16;
+    static constexpr size_t kDefaultBufferedChunks = 64;
+
+    // Tunning the max_buffer_chunks according to requested limit
+    static constexpr size_t tunning_buffered_chunks(int limit) {
```

Comment:
would better add more comment here. let others know how do we get this value and  this heuristic value is key for performance.

Reply:
done.